client/src/components/Workflow/Editor/Attributes.vue

```diff
@@ -105,7 +104,7 @@ export default {
             default: null,
         },
         parameters: {
-            type: LegacyParameters,
```

Comment:
Seems like losing the type information here for 'Object' is a step backwards -- if we're doing this can you add a validator instead?

Reply:
Thanks for catching that.